tests/integration-package/lambda-files.tests.js

```diff
@@ -89,4 +90,13 @@ describe('Integration test - Packaging', function() {
       .then(() => listZipFiles(path.join(cwd, '.serverless/hello2.zip')))
       .then(zipfiles => expect(zipfiles).to.deep.equal(['handler2.js']));
   });
+
+  it('handles package individually on function level with include/excludes correctly', () => {
```

Comment:
@mthenw I've run this on `master` and it also passed. Do you know why? Can we make a test here, that will test the specific case we're fixing here?

Reply:
Yes, I also noticed that and my understanding is that package step/cmd work fine. The issue is that it changed the config causing issue in deploy step/cmd.

Reply:
Ok, got it!

References to zip files were messed up, but not zip files on their own. In that case it might good to reuse this fixture and add test to https://github.com/serverless/serverless/blob/91ae8bcc17d3ab7874507c8192375b4a28627592/tests/integration-package/cloudformation.tests.js#L85 (as I take this is were it'll pick the issue on `master`)

Reply:
I will double-check that but I don't think that this bug influence CF template anyhow. When I wrote "the config" in my previous comment I meant internal JS representation on serverless.yaml (`this.serverless.service`).

Reply:
Yeah, I checked it by adding integration tests using `individually-function` fixtures and it passes both on master and my branch.

Reply:
So it's a purely pre-deploy validation logic error?

Error suggests that serverless is unable to locate prepared zip file. Still from what you're saying:
- File of expected name is actually created as expected (and that's validated by `lambda-files.tests.js)
- File is correctly referenced in generated CloudFormation tamplate (that could be validated in `cloudformation.tests.js`)

In other words: On `master` if this error would be muted the deployment will be successful and as expected, right?

Reply:
> So it's a purely pre-deploy validation logic error?

Yes. That's my understanding of it. 

> In other words: On master if this error would be muted the deployment will be successful and as expected, right?

I haven't tried that but it might be true.

Reply:
Ok, I looked into it more deeply, issue is that by overriding `functionObject.package` here: https://github.com/serverless/serverless/blob/70446d94ede449e5081211d89ec23373af02dc1a/lib/plugins/package/lib/packageService.js#L166
We've actually overriden existing package config that contained `individually: true`.

So effect was that `individually: true` was gone from the config , and deploy validation was crashing, as it was searching for project wide artifact for given function

Reply:
Yes exactly